app/assets/stylesheets/accounts.scss

```diff
@@ -188,3 +152,8 @@ $_current-indicator-width: 4px;
     padding-left: 30%;
   }
 }
+
+// this is a temporary override while we figure out the slightly strange styling of this component
+.gem-c-summary-list {
+  border: 0;
```

Comment:
Not sure if I'm missing something but this rule doesn't seem to change anything for me? 🤔

Reply:
You're right, it doesn't. Well, it doesn't anymore, now that I've wrapped the summary lists in that spacing wrapper. The reason it was in originally was to counter the slightly odd CSS rules the component has for when there are multiple instances of it in succession, which added an extra border to this and made it look wrong. But as you said, it's now not needed, so I've taken it out.

https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/stylesheets/govuk_publishing_components/components/_summary-list.scss#L13

Reply:
Ooooh, yep I see 👍  that makes sense, thanks for clarifying.